front/src/assets/compositionCode/compositionCodes.ts

```diff
@@ -0,0 +1,21 @@
+const COMPOSITION_CODES = [
```

Comment:
Should we somehow name this file in a way that makes it clear it's for STDCM specifically?

Reply:
I think this list could be used elsewhere

Reply:
The list is very specific to STDCM I believe?

Reply:
Yes it's for stdcm

Reply:
we could rename it `stdcmCompositionCodes` ?

Reply:
I can, @emersion, you think it's better ?

Reply:
Yes, I think putting "stdcm" in the filename is a good idea.

Reply:
Done